modules/islandora_crossref_citations/README.md

```diff
@@ -0,0 +1,49 @@
+# Islandora Crossref Citations
+
+## Introduction
+
+Islandora Crossref Citations generates a block which displays citation counts for any object with a DOI, via the Crossref Cited-by service, for use with Islandora Badges.
+
+The badge will only display on objects with a MODS datastream and a DOI (Digital Object Identifier).
+
+## Requirements
+
+This module requires the following modules/libraries:
+
+* [Islandora](https://github.com/islandora/islandora)
+* [Islandora Scholar](https://github.com/Islandora/islandora_scholar)
```

Comment:
If Scholar is a dependency, why is it not part of the .info file? Is that so in every other submodule?